packages/esm-patient-search-app/src/patient-search-page/patient-banner/contact-details/relationships.resource.tsx

```diff
@@ -0,0 +1,90 @@
+import { openmrsFetch } from '@openmrs/esm-framework';
+import useSWR from 'swr';
+
+const customRepresentation =
```

Comment:
This is such a wild API...

Reply:
Yes it is 😆

Reply:
Relationships really are one of the weirdest bits of modelling in OpenMRS... In a way it's too flexible to actually be valuable (many real-world implementations simply use a patient attribute or a obs to store the actual related-person contact details... so make sure this is swappable or hideable).

Reply:
Sure @ibacher , but I'll make these changes in the next iteration.
Merging this one at the moment.